.github/actions/openziti-tunnel-build-action/redhat-8/entrypoint.sh

```diff
@@ -39,16 +39,17 @@ done
 cmake -E make_directory ./build
 (
     [[ -d ./build ]] && rm -r ./build
-    cmake -E make_directory ./build  
+    cmake -E make_directory ./build
     # allow unset for scl_source scripts
     set +u
     source scl_source enable gcc-toolset-10 \
         && cmake \
             --preset "${cmake_preset}" \
             -DCMAKE_BUILD_TYPE="${cmake_config}" \
             -DBUILD_DIST_PACKAGES=ON \
+            -DVCPKG_OVERLAY_PORTS="./vcpkg-overlays/linux-syslibs/default;./vcpkg-overlays/linux-syslibs/redhat8" \
```

Comment:
I'd rather not combine the overlay directories as I think it can lead to surprises.

The intent with the "default" overlay directory is that it contains all of the packages that can be provided by the system for modern distros (e.g. the "ideal" case), and the distro-specific overlay directories exist for platforms that can't use one or more of the packages from "default".

Reply:
The alternative is to replicate the default in each of those directories. Simply specifying the single overlay directory overrides the default for the preset altogether. This is what led me to wonder if our current implementation is working as intended for other places where this is specified.